diff --git a/node_modules/expo-dev-launcher/ios/SwiftUI/CrashReportView.swift b/node_modules/expo-dev-launcher/ios/SwiftUI/CrashReportView.swift
index dd98f37..50aa878 100644
--- a/node_modules/expo-dev-launcher/ios/SwiftUI/CrashReportView.swift
+++ b/node_modules/expo-dev-launcher/ios/SwiftUI/CrashReportView.swift
@@ -35,7 +35,9 @@ struct CrashReportView: View {
       }
       .padding()
     }
+    #if !os(tvOS)
     .background(Color(.systemBackground))
+    #endif
     .navigationBarHidden(true)
   }
 
@@ -128,7 +130,9 @@ struct CrashReportView: View {
         .frame(maxWidth: .infinity, alignment: .leading)
       }
       .frame(maxHeight: 200)
+      #if !os(tvOS)
       .background(Color(.secondarySystemGroupedBackground))
+      #endif
       .cornerRadius(8)
     }
     .frame(maxWidth: .infinity, alignment: .leading)
diff --git a/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift b/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift
index 74d0c4c..d705ab5 100644
--- a/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift
+++ b/node_modules/expo-dev-launcher/ios/SwiftUI/HomeTabView.swift
@@ -64,7 +64,9 @@ struct HomeTabView: View {
         .padding()
     }
     .buttonStyle(PlainButtonStyle())
+    #if !os(tvOS)
     .background(Color(.secondarySystemGroupedBackground))
+    #endif
     .cornerRadius(18)
   }
 }
