diff --git a/node_modules/expo-router/build/link/preview/native.js b/node_modules/expo-router/build/link/preview/native.js
index de5e535..aba2ade 100644
--- a/node_modules/expo-router/build/link/preview/native.js
+++ b/node_modules/expo-router/build/link/preview/native.js
@@ -7,7 +7,7 @@ exports.NativeLinkPreview = NativeLinkPreview;
 exports.NativeLinkPreviewContent = NativeLinkPreviewContent;
 const expo_1 = require("expo");
 const react_native_1 = require("react-native");
-const areNativeViewsAvailable = process.env.EXPO_OS === 'ios' && global.RN$Bridgeless === true;
+const areNativeViewsAvailable = process.env.EXPO_OS === 'ios' && !react_native_1.Platform.isTV && global.RN$Bridgeless === true;
 const LinkPreviewNativeActionView = areNativeViewsAvailable
     ? (0, expo_1.requireNativeView)('ExpoRouterNativeLinkPreview', 'LinkPreviewNativeActionView')
     : null;
